misc/atrium-specs/src/commonMain/kotlin/ch/tutteli/atrium/specs/integration/AbstractCharSequenceToContainAtLeastExpectationsTest.kt

```diff
@@ -0,0 +1,361 @@
+import ch.tutteli.atrium.api.verbs.internal.expect
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.specs.*
+import ch.tutteli.atrium.testfactories.TestFactory
+import ch.tutteli.atrium.translations.DescriptionCharSequenceExpectation
+
+abstract class AbstractCharSequenceToContainAtLeastExpectationsTest(
+    private val toContainAtLeastPair: Pair<(String, String) -> String, Fun3<CharSequence, Int, Any, Array<out Any>>>,
+    private val toContainAtLeastIgnoringCasePair: Pair<(String, String) -> String, Fun3<CharSequence, Int, Any, Array<out Any>>>,
+    private val toContainAtLeastButAtMostPair: Pair<(String, String, String) -> String, Fun4<CharSequence, Int, Int, Any, Array<out Any>>>,
+    private val toContainAtLeastButAtMostIgnoringCasePair: Pair<(String, String, String) -> String, Fun4<CharSequence, Int, Int, Any, Array<out Any>>>,
+    private val notToContainPair: Pair<String, (Int) -> String>,
+    private val exactlyPair: Pair<String, (Int) -> String>,
+    private val errorMsgAtLeastButAtMost: (Int, Int) -> String,
+) : ExpectationFunctionBaseTest() {
+
+    val toContainAtLeastSpec = toContainAtLeastPair.second
+    val toContainAtLeastIgnoringCaseSpec = toContainAtLeastIgnoringCasePair.second
+    val toContainAtLeastButAtMostSpec = toContainAtLeastButAtMostPair.second
+    val toContainAtLeastButAtMostIgnoringCaseSpec = toContainAtLeastButAtMostIgnoringCasePair.second
+
+    fun subjectLessTest() = subjectLessTestFactory(
+        toContainAtLeastSpec.forSubjectLessTest(1, "2.3", arrayOf()),
+        toContainAtLeastIgnoringCaseSpec.forSubjectLessTest(1, 'a', arrayOf()),
+        toContainAtLeastButAtMostSpec.forSubjectLessTest(1, 2, "aA", arrayOf()),
+        toContainAtLeastButAtMostIgnoringCaseSpec.forSubjectLessTest(1, 2, 2.3, arrayOf())
+    )
+
+    fun Expect<CharSequence>.toContainAtLeastFun(atLeast: Int, a: Any, vararg aX: Any) =
+        toContainAtLeastSpec(this, atLeast, a, aX)
+
+    fun Expect<CharSequence>.toContainAtLeastIgnoringCaseFun(atLeast: Int, a: Any, vararg aX: Any) =
+        toContainAtLeastIgnoringCaseSpec(this, atLeast, a, aX)
+
+    fun Expect<CharSequence>.toContainAtLeastButAtMostFun(atLeast: Int, atMost: Int, a: Any, vararg aX: Any) =
+        toContainAtLeastButAtMostSpec(this, atLeast, atMost, a, aX)
+
+    fun Expect<CharSequence>.toContainAtLeastButAtMostIgnoringCaseFun(
+        atLeast: Int, atMost: Int, a: Any, vararg aX: Any
+    ) = toContainAtLeastButAtMostIgnoringCaseSpec(this, atLeast, atMost, a, aX)
+
+    val valueWithIndent = "$indentRootBulletPoint$listBulletPoint$value"
+
+    @TestFactory
+    fun toContainAtLeast__subject_throws_an_IllegalArgumentException() =
+        testFactory(toContainAtLeastSpec) { toContainAtLeastFun ->
+            val (notToContain, errorMsgNotToContain) = notToContainPair
+
+            it("for at least -1 -- only positive numbers") {
+                ch.tutteli.atrium.api.verbs.internal.expect {
```

Comment:
```suggestion
                expect {
```
replace all usages of ch.tutteli.atrium.api.verbs.internal.expect in this file (we want to use the expect of ExpectTestExecutableForTests which in turn uses `ExpectTestExecutable.expect` otherwise the expect will not be in the group (in the js target) but standalone, breaking the flow